main.go

```diff
@@ -1147,22 +1150,12 @@ var exportCmd = &cobra.Command{
 		var out io.Writer
 		if len(args) > 0 {
 			filename := args[0]
-			var f *os.File
-			f, err = os.OpenFile(filename, os.O_WRONLY|os.O_CREATE|os.O_EXCL, 0440)
+			file, err := os.OpenFile(filename, os.O_WRONLY|os.O_CREATE, 0600)
 			if err != nil {
 				return err
 			}
-			defer func() {
-				if errc := f.Close(); err == nil && errc != nil {
-					err = errc
-				}
-				if err != nil {
-					os.Remove(filename)
-				} else {
-					fmt.Printf("Export finished successfully in file %q.\n", filename)
-				}
-			}()
-			out = f
+			defer file.Close()
```

Comment:
It is a bad practice to not check if `file.Close()` has returned an error. Why haven't you kept the previous code?

Reply:
Previous code was quite buggy with all or nothing closing and some other error corner cases.
Will add a check on this.

Reply:
The previous code on this file looks correct. Can you be more specific on what were the issues?

Reply:
Was another part now removed, with 3 cascading "all or nothing" defer. This is fine.

Reply:
But the error on `file.Close()` here is still not checked!!